src/builder.rs

```diff
@@ -79,7 +79,7 @@ impl Builder {
     /// # Panics
     /// Panics if the provided `filter` regular expression is longer than [`u32::MAX`] bytes.
     #[must_use]
-    pub fn config_paths_count(&self, filter: Option<&'_ str>) -> u32 {
+    pub fn config_paths_count(&mut self, filter: Option<&'_ str>) -> u32 {
```

Comment:
Though the docs say:

>  * @note This function is not thread-safe and the memory of the paths object must be freed by the caller.

This is likely misdocumentation from a previous iteration where you didn't have to free the result. You can check with @Anilm3 once he's back though